extensions/web-tracker/wab/src/main/resources/META-INF/cxs/schemas/identify.json

```diff
@@ -13,12 +13,7 @@
   "allOf": [{ "$ref": "https://unomi.apache.org/schemas/json/event/1-0-0" }],
   "properties" : {
     "properties" : {
-      "type" : "object",
-      "properties" : {
-        "consent" : {
-          "$ref" : "https://unomi.apache.org/schemas/json/consent/1-0-0"
-        }
-      }
+      "$ref" : "https://unomi.apache.org/schemas/json/events/identify/properties/1-0-0"
     },
     "source" : {
       "$ref" : "https://unomi.apache.org/schemas/json/customitems/page/1-0-0"
```

Comment:
Does this event should not have his own JSON Schemas for the target/source object ?

Reply:
In the example it is by default using the site as a source and a page as a target but in the schema it was not correct. I'll fix that.